wcomponents-core/src/main/java/com/github/bordertech/wcomponents/AbstractWComponent.java

```diff
@@ -1019,46 +939,29 @@ private static boolean isFlagSet(final int flags, final int mask) {
 		return isSet;
 	}
 
-	/**
-	 * {@inheritDoc}
-	 * @deprecated 1.4 causes a11y issues, no replacement.
-	 */
 	@Deprecated
 	@Override
 	public boolean hasTabIndex() {
 		return false;
 	}
 
-	/**
```

Comment:
Please retain this JavaDoc comment as the inheritance needs to be explicit due to the other comments in the block.